tests/conftest.py

```diff
@@ -1,17 +1,60 @@
+import base64
 import os
 
 import pytest
+from six.moves.urllib.parse import parse_qs, urlparse
 
-from wptserve import server
+from wptserve import (
+    handlers,
+    request,
+    routes as default_routes,
+    server,
+)
+
+class Authentication(request.Authentication):
```

Comment:
I would like to avoid that. So lets see if James can release a new version of wptserve on PyPI. So we would only have to update the dependency instead.